app/src/docs/v1.api-spec.yaml

```diff
@@ -17,30 +17,348 @@ security:
   - BearerAuth: []
     OpenID: []
 paths:
-  /hello:
+  '/':
     get:
-      summary: Returns hello world result
-      operationId: getHello
+      summary: List all user accessible objects
+      description: >-
+        This endpoint returns a list of all user accessible objects.
+      operationId: GetAccessibleObjects
+      tags:
+        - Object
```

Comment:
We'll need to talk about how we're tagging all of these endpoints later once we have something more solid as IIRC tags affect how they're clustered/grouped together on ReDoc/Swagger renders.